src/libraries/System.Private.CoreLib/src/System/SpanHelpers.Byte.cs

```diff
@@ -194,7 +195,7 @@ public static unsafe int IndexOf(ref byte searchSpace, byte value, int length)
             nuint offset = 0; // Use nuint for arithmetic to avoid unnecessary 64->32->64 truncations
             nuint lengthToExamine = (nuint)(uint)length;
 
-            if (Avx2.IsSupported || Sse2.IsSupported)
+            if (Avx2.IsSupported || Sse2.IsSupported || AdvSimd.IsSupported)
```

Comment:
Somewhat related, but now that `ARM64` is hitting this path, we won't ever hit `Vector.IsHardwareAccelerated` for CoreCLR.

I think that might just leave Mono using `Vector.IsHardwareAccelerated` so we might be able to remove this soon?

Reply:
Perhaps. I will leave it for another PR.